automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestParameterSetting.scala

```diff
@@ -206,12 +203,155 @@ class TestParameterSetting  extends ExtractionTest {
   val t1c = "We therefore assume that S(0) = 6.8 – 0.5 = 6.3 million."
   failingTest should s"extract the parameter setting(s) from t1c: ${t1c}" taggedAs(Somebody) in {
     val desired = Seq(
-      "S(0)" -> Seq("6.3") // fixme: is million a param setting or unit?
+      "S(0)" -> Seq("6.3 million")
     )
     val mentions = extractMentions(t1c)
     testParameterSettingEvent(mentions, desired)
   }
 
+  // SuperMaaS tests
+
+  val u1a = "Default nitrogen removal is 15 % for sheep & beef, and 25 % for dairy grazed pastures but it remains the user's choice to set a sensible value for their system ."
+  failingTest should s"extract the parameter setting(s) from u1a: ${u1a}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "nitrogen removal" -> Seq("15"), //fixme: handle percentages
+      "nitrogen removal" -> Seq("25")
+    )
+    val mentions = extractMentions(u1a)
+    testParameterSettingEvent(mentions, desired)
+  }
+
+  val u2a = "It shows that the pasture is not harvested before 1/07/1995 , the harvest frequency is once every 21 days and the harvest residual is 1250 kg / ha ."
+  failingTest should s"extract the parameter setting(s) from u2a: ${u2a}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "harvest residual" -> Seq("1250") //todo: add this as unit test, too
+    )
+    val mentions = extractMentions(u2a)
+    testParameterSettingEvent(mentions, desired)
+  }
+
+  val u3a = "85 % of grazed / harvested nitrogen returns to soil , of which 60 % is in urine and returned to soil profile to a depth of 300 mm ."
+  failingTest should s"extract the parameter setting(s) from u3a: ${u3a}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "soil profile to a depth" -> Seq("300") //fixme: should actually be soil profile depth but this is good enough
+    )
+    val mentions = extractMentions(u3a)
+    testParameterSettingEvent(mentions, desired)
+  }
+
+  val u4a = "where sowing depth was known and it is set to 1.5 o Cd per mm ."
+  passingTest should s"extract the parameter setting(s) from u4a: ${u4a}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "sowing depth" -> Seq("1.5")
+    )
+    val mentions = extractMentions(u4a)
+    testParameterSettingEvent(mentions, desired)
+  }
+
+  val u4atoy = " where St is sowing depth  and it is set to 1.5 o Cd per mm ."
+  passingTest should s"extract the parameter setting(s) from u4atoy: ${u4atoy}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "St" -> Seq("1.5")
+    )
+    val mentions = extractMentions(u4atoy)
+    testParameterSettingEvent(mentions, desired)
+  }
+
+  // todo: add a units test
```

Comment:
Again, if you wanted this to be addressed within this PR, please do so before you merge this!